packages/generator-adp/src/translations/generator-adp.i18n.json

```
@@ -33,6 +37,8 @@
         "projectFolderPath": "Project Path",
         "ui5VersionLabel": "SAPUI5 Version",
         "ui5VersionTooltip": "Select the SAPUI5 version you want to use to preview your app variant.",
+        "addDeployConfig": "Add deployment configuration",
```

```suggestion
        "addDeployConfig": "Add deployment configuration",
```
Is this a title,  a button, placeholder?

---

Thanks. Updated suggestion.